README.md

```diff
@@ -2,3 +2,122 @@ ASP.NET Web Sdk targets
 ======================
 
 ASP.NET Web Sdk targets contains the tasks, targets and packages required to build and publish Web Applications.
+
+Publish Commandline Usage:
+
+1. Folder publish without a profile (Default Publish):
+    - msbuild WebApplication.csproj /p:DeployOnBuild=true /p:PublishUrl="C:\deployedApp\newapp"
+    or
```

Comment:
Not sure how to fix it, but things like this are being included in the previous bullet